core/src/main/java/org/infinispan/commands/functional/ReadOnlyKeyCommand.java

```diff
@@ -42,6 +66,7 @@ public void writeTo(ObjectOutput output) throws IOException {
       output.writeObject(key);
       output.writeObject(f);
       Params.writeObject(output, params);
+      output.writeObject(encodingClasses);
```

Comment:
Both wrappers serialization could be encoded in a single byte; each encoder would be another byte, so a total overhead of 3 bytes would be added per command